manifests/agent.pp

```diff
@@ -187,10 +187,17 @@
   $audit_rules                       = $wazuh::params_agent::audit_rules,
 
   # active-response
-  $ossec_active_response_disabled          =  $wazuh::params_agent::active_response_disabled,
-  $ossec_active_response_linux_ca_store    =  $wazuh::params_agent::active_response_linux_ca_store,
-  $ossec_active_response_windows_ca_store  =  $wazuh::params_agent::active_response_windows_ca_store,
-  $ossec_active_response_ca_verification   =  $wazuh::params_agent::active_response_ca_verification,
+  $ossec_active_response_disabled             =  $wazuh::params_agent::active_response_disabled,
+  $ossec_active_response_linux_ca_store       =  $wazuh::params_agent::active_response_linux_ca_store,
+  
+  $ossec_active_response_ca_verification      =  $wazuh::params_agent::active_response_ca_verification,
+  $ossec_active_response_command              =  $wazuh::params_manager::active_response_command,
```

Comment:
Why using a parameter from `params_manager`?

Reply:
It's wrong, it should be `params_agent`. Thanks for reporting. 
https://github.com/wazuh/wazuh-puppet/pull/234/commits/701a7e45f8999b127b440c425eabafd3cda50f8c